Document available URL parameters for Sandbox (#333)
apollographql/docs: src/content/graphos/explorer/sandbox.mdx

See [this section](./operation-collections/#sandbox-collections).

## URL parameters

You can customize a few things about Sandbox when first loading the page via URL parameters—this enables users to skip a few steps when loading Sandbox from a link. These are all the parameters available:

| Parameter   | Description                                                                                                                  |
|-------------|------------------------------------------------------------------------------------------------------------------------------|
| `endpoint`  | The URL of the GraphQL endpoint that Sandbox introspects and sends requests to. The default value is `http://localhost:4000` |

> | `document`  | The GraphQL query that should be populated in the operation panel                                                            |

Reviewer: ```suggestion
| `document`  | A GraphQL operation to populate in the Explorer's editor on load.                                                     |
```

Reviewer: (I'm stealing these replacement descriptions from the similar table in the "Embedding the Explorer" article)

> | `variables` | The variables for the query that should be set below the operation panel, must be a JSON object.                             |

Reviewer: ```suggestion
| `variables` | A serialized JSON object containing initial variable values to populate in the Explorer on load.                             |
```

> | `headers`   | Headers that should be included in requests to the `endpoint`, populated below the operation panel, must be a JSON object.   |

Reviewer: ```suggestion
| `headers`   | A serialized JSON object containing initial HTTP header values to populate in the Explorer on load.   |
```

For example, the following URL sets each of these parameters:

```
https://studio.apollographql.com/sandbox/explorer?endpoint=http://localhost:4200&document=query hello($name: String!) { hello(name: $name)}&variables={"name": "Sandbox User"}&headers={"Cache-Control": "no-cache"}
```

> Note that, for readability, the above URL is not URL-encoded. Most browsers will automatically encode the URL for you, but it is best practice to URL-encode the parameters before providing them to a user.

> This URL results in Sandbox loading like the following screenshots:

Reviewer: ```suggestion
The example URL above opens Sandbox with the following initial state:
```
